src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -69,6 +70,8 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.updateSortedPersonList(DEFAULT_PERSON_COMPARATOR);
         this.sortedMeetings = new SortedList<>(this.addressBook.getMeetingList(), Comparator.naturalOrder());
         this.sortedReminders = new SortedList<>(this.addressBook.getReminderList(), Comparator.naturalOrder());
+        this.filteredReminders = new FilteredList<>(sortedReminders);
```

Comment:
P.S. for this section, is it possible to change such that you create a sorted list from the filtered list? So that it's more consistent with how the other models behave since the other models also filter first, then sort.

Reply:
Hmmmm I've tried this approach, but seems edit and delete command only work if the filtering is done on the sorted list.